admin/class-admin-apple-settings.php

```diff
@@ -154,11 +154,13 @@ public function register_assets( $hook ) {
 			'../vendor/select2/select2.min.css', array() );
 		wp_enqueue_style( 'apple-news-settings-css', plugin_dir_url( __FILE__ ) .
 			'../assets/css/settings.css', array() );
+		wp_enqueue_style( 'jquery-ui-smoothness',
+			'https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css', array() );
```

Comment:
Any benefit to including this with the theme rather than loading an external resource?